nixos/doc/manual/development/writing-nixos-tests.section.md

```diff
@@ -98,10 +98,10 @@ There are a few special NixOS options for test VMs:
 
 `virtualisation.writableStore`
 
-:   By default, the Nix store in the VM is not writable. If you enable
-    this option, a writable union file system is mounted on top of the
-    Nix store to make it appear writable. This is necessary for tests
-    that run Nix operations that modify the store.
+:   By default, the Nix store in the VM is writable - a writable union
+    file system is mounted on top of the shared host Nix store to make
+    it appear writable. This is necessary for tests that run Nix
+    operations that modify the store.
```

Comment:
Note that in a02bb00156086b45e68c1112008db506734f8649 the default was changed to `true`, with a `FIXME` comment. I have updated some docs to be consistent with that. Is there a convention so that when/if it gets changed back we will remember to update the docs again?